grammars/fstar.cson

```diff
@@ -70,7 +70,7 @@
   }
   { # Keywords
     'name'  : 'keyword.control.fstar'
-    'match' : '\\b(assert|in|if|then|else|fun|try|function|match|with|and)\\b'
+    'match' : '\\b(assert|in|if|then|else|fun|λ|try|function|match|with|and)\\b'

```

Comment:
Is `λ` a valid F* keyword now?

Reply:
The [unicode lexer](https://github.com/FStarLang/FStar/wiki/Unicode-Support-in-F%E2%98%85) accepts `λ` as as a synonym for `fun`. 

eg. `let id = λ x -> x`

Reply:
Thanks for the clarification!